programs/zstdcli.c

```diff
@@ -452,6 +465,38 @@ static void printVersion(void)
 #endif
 }
 
+/* Environment variables for parameter setting */
+#define ENV_CLEVEL "ZSTD_CLEVEL"
+
+/* functions that pick up environment variables */
+int init_cLevel() {
```

Comment:
minor : prefer declaration `static int init_cLevel(void)`

Reply:
Sorry, forgot that one.